package.json

```diff
@@ -1,83 +1,32 @@
 {
-  "name": "docusign",
-  "version": "8.0.0",
-  "description": "A DocuSign API helper library with promise support",
-  "main": "docusign.js",
+  "name": "docusign-esign",
+  "version": "2.0.0",
+  "description": "DocuSign Node.js API client.",
+  "license": "LicenseRef-LICENSE",
```

Comment:
LicenseRef-LICENSE means that we are publishing the package under a custom license which is specified in the attached file LICENSE.
@Ergin008 should we turn to MIT already?